packages/core-flows/src/product/workflows/create-products.ts

```diff
@@ -1,7 +1,14 @@
 import { ProductTypes } from "@medusajs/types"
-import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
-import { createProductsStep } from "../steps"
+import {
+  WorkflowData,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { createProductsStep, createVariantPricingLinkStep } from "../steps"
+import { createPriceSetsStep } from "../../pricing"
 
+// TODO: We should have separate types here as input, not the module DTO. Eg. the HTTP request that we are handling
```

Comment:
This is what I did in my PR and found the experience to be much better working with modules.

Reply:
I'll just add some TODOs for now, there is a lot of cleanup to be done anyway